### ISSUE TITLE
Add warning when file set for USE_MDFILE_AS_MAINPAGE isn't found

### ISSUE THREAD

**Issue by dereckson** [doxygen/doxygen#10015]
**Describe the bug**

If the file set for USE_MDFILE_AS_MAINPAGE isn't found, for example because it's not set in INPUT, there is no warning in doxygen output.

**Expected behavior**
warning: README.md not found, but set in USE_MDFILE_AS_MAINPAGE

**To Reproduce**

```
mkdir -p test/src
cd test
touch README.md
cat > Doxyfile
INPUT = src
USE_MDFILE_AS_MAINPAGE = README.md
```

**Version**
1.9.6

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #10022

**Comment by albert-github** [comment]
Code has been integrated in master on GitHub (please don't close the issue as this will be done at the moment of an official release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.7.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
